DER/DerivativesContracts/Forwards.rdf

```diff
@@ -260,113 +434,169 @@
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasMonetaryAmount"/>
+		<rdfs:label xml:lang="en">has additional costs</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
-		<skos:definition xml:lang="en">The forward price of the underlying.</skos:definition>
+		<skos:definition xml:lang="en">indicates costs, such as up front costs, brokerage fees and the like, that must be paid on delivery</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;hasBuyer">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;hasCounterparty"/>
 		<rdfs:label xml:lang="en">has buyer</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardContract"/>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;Forward"/>
 		<rdfs:range rdf:resource="&fibo-der-drc-bsc;PayingParty"/>
 	</owl:ObjectProperty>
 	
 	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;hasContractSize">
 		<rdfs:label xml:lang="en">has contract size</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardContract"/>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;Forward"/>
 		<rdfs:range rdf:resource="&xsd;integer"/>
 		<skos:definition xml:lang="en">indicates the deliverable quantity of goods, commodities, or shares underlying the futures contract. This is denominated as a multiple of the lot size in the case of a commodity future. For financial futures this is the number of units of the underlying security.</skos:definition>
 	</owl:DatatypeProperty>
 	
+	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;hasConversionFactor">
+		<rdfs:label xml:lang="en">has conversion factor</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;BondFuture"/>
+		<rdfs:range rdf:resource="&xsd;decimal"/>
+		<skos:definition xml:lang="en">indicates the price of the delivered bond/note ($1 par value) to yield a fixed rate. The conversion factor is used to calculate a final delivery price.</skos:definition>
+	</owl:DatatypeProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;hasForwardUnderlierPrice">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasPrice"/>
+		<rdfs:label xml:lang="en">has forward underlier price</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+		<skos:definition xml:lang="en">indicates the price of the underlier of the forward as specified in delivery terms</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;hasLotSize">
+		<rdfs:label xml:lang="en">lot size</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;CommodityFuture"/>
+		<rdfs:range rdf:resource="&fibo-fnd-qt-qtu;QuantityValue"/>
+		<skos:definition xml:lang="en">How many of the underlying commodity can be sold. The groupings of certain units of measure in the underlying commodity.</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;hasMultiple">
```

Comment:
seems like it should be a more general property. If not give it a more specific name